Code/STAGE1get_that_wfdb.m

```diff
@@ -0,0 +1,56 @@
+%% Jericho Lawson
+%% Summer 2019, 2021
+%% Extracting AFib Data
+
+% Using the mcode from the training2017 folder of the training2017.zip 
+% file, the code extracts annotations from the 2017 PhysioNet/CinC 
+% challenge and generates basic RR-interval information for each of 
+% the subjects.
+
+% Directory where all files are located
+SOURCE_FROM = "/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing/training2017";
```

Comment:
We need to SOURCE_FROM in every doc

Reply:
I'll double-check that I added those suggestions in there. I updated the first two stages of code to make the process of changing file names/globals smoother. I'm still in the process of refining the code, but the functionality should remain the same in the updated code I just committed.